integration.Dockerfile

```
@@ -0,0 +1,35 @@
+FROM fedora:29
+LABEL \
+    name="Operators Manifests Push Service" \
+    vendor="Red Hat, Inc" \
+    image="integration" \
+    license="GPLv3"
+
+# Test and build dependencies.
+RUN dnf -y install \
```

```suggestion
RUN dnf -y install --setopt=tsflags=nodocs --setopt=install_weak_deps=false \
```
The above would make the resulting image smaller, by not installing documentation and weak dependencies.